src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="/images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <div class="card__code">
```

Comment:
```suggestion
      >
      
      <h2 class="card__name">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </h2>
      
      <div class="card__code">
```
делай отступы для лучшей читаемости кода